tests/test_sum_zintox.f90

```diff
@@ -0,0 +1,103 @@
+program test_sum_zintox
```

Comment:
because there is only very few differences between this test and test_sum_yintox, maybe they can be combined into a single test (will just ease maintenance if/when things change)

Reply:
Good point - done